docs/guide.md

```diff
@@ -300,6 +300,22 @@ func injectFooBar() FooBar {
 
 And similarly if the injector needed a `*FooBar`.
 
+Sometimes it is useful to restrict certain fields to be accidentally filled by
```

Comment:
Wordsmithing:

```
It is sometimes useful to prevent certain fields from being filled in
by the injector, especially when passing `*` to `wire.Struct`. You
can tag a field with `` `wire:"-"` `` to have Wire ignore such fields.
For example:
```

Reply:
Done.